### DISCUSSION TITLE
How to set custom response header while serving a file under public directory

### DISCUSSION THREAD

**Discussion by bernard-ng** [symfony/symfony#43014] [Q&A] [answered]
Is there anyway to add a response header while serving a particular asset file ?

supposed that I want to add the header "Service-Worker-Allowed: /" while serving a file under the public directory like "/build/sw.js"

how can i do that ?

**Comment by lyrixx** [comment, marked as answer]
If PHP is not involved, you can do that with you webserver (nginx, apache)
